### PR TITLE
Add compas_rrc package (online robotic control for COMPAS framework)

### PR DESCRIPTION
**Pull request by gonzalocasas**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gonzalocasas** [comment]
@conda-forge/help-python perhaps you could point me in the right direction here. The linux build is failing because some shared lib does not seem to be present. The shared lib in question is a `.so` file that is part of `compas_fab`. I remember that when I created the recipe for it, I had similar issues, and the fix ended up being `binary_relocation: false`, but now I tried with and without that, but no success. Any hints?

**Comment by gonzalocasas** [comment]
This is ready for review. @conda-forge-admin, please ping team

File: recipes/compas_rrc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+about:
+  home: https://github.com/compas-rrc/compas_rrc
+  summary: 'COMPAS RRC: Online control for ABB robots over a simple-to-use Python interface.'
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - gonzalocasas
+    - fleischp
```
Comment (ocefpaf) [inline recipes/compas_rrc/meta.yaml:44]: @fleischp are you OK becoming a maintainer here?

Comment (fleischp) [reply recipes/compas_rrc/meta.yaml:44]: that is fine for me.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**